internal/platform/fetcher/bilibili.go

```
@@ -0,0 +1,247 @@
+		return f.fetchDynamics(ctx, source)
+	}
+	return f.fetchVideos(ctx, source)
+}
+
+// fetchVideos handles video list APIs (popular, space/search, ranking).
+func (f *BiliBiliFetcher) fetchVideos(ctx context.Context, source Source) ([]Item, error) {
+	body, err := fetchBody(ctx, f.client, source.URL)
+	if err != nil {
+		return nil, err
+	}
+	defer body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `body.Close` is not checked

(errcheck)